packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -164,20 +155,9 @@ export default {
             commit, state, dispatch, rootGetters
         }, { url, data, timeout }) => {
             // TODO: deal with exceptions and handle this action properly (when the functionality is ready)
```

Comment:
Is there a ticket for this?  If there is. could we reference it?

Reply:
I'm actually not sure on this one - @SachaZvetelman do you know?

Reply:
No, we cannot reference tickets on this project, I'm afraid. Ping me if you want a more thorough explanation :)
However, this comment seem to be out of date now, since we're handling exceptions.